.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### PR Checklist
+- [ ] Have you updated the required documentation based on the current PR
+  changes
+- [ ] Have you written new tests for your core changes, as applicable?
+
+**Commit Formatting**
+
+Please follow the `seven rules of a great commit message`
```

Comment:
Should this be a link to some article somewhere?  If so, I'd use _italics_ as the text type - rather than `code`.

Reply:
no link, the seven rules are below the text mentioned above